tests/Application/src/Tests/Controller/SpeakerUiTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Tests\Controller;
+
+use ApiTestCase\ApiTestCase;
+use App\Conference\Entity\Speaker;
+use App\Conference\Factory\SpeakerFactory;
+use Coduo\PHPMatcher\Backtrace\VoidBacktrace;
+use Coduo\PHPMatcher\Matcher;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
+
+final class SpeakerUiTest extends ApiTestCase
+{
+    use Factories;
+    use ResetDatabase;
+
+    /** @test */
+    public function it_allows_browsing_speakers(): void
+    {
+        SpeakerFactory::new()
+            ->withFirstName('Francis')
```

Comment:
Can Francis sue sylius for becoming test data? :joy:

Reply:
He's already on Sylius stack phpunit tests where this test is inspired.